src/solace_ai_connector/solace_ai_connector.py

```diff
@@ -98,15 +100,6 @@ def wait_for_flows(self):
                 self.stop()
                 self.cleanup()
 
-    def stop(self):
-        """Stop the Solace AI Event Connector"""
-        log.info("Stopping Solace AI Event Connector")
-        self.stop_signal.set()
-        self.timer_manager.stop()  # Stop the timer manager first
-        self.wait_for_flows()
-        if self.trace_thread:
-            self.trace_thread.join()
-
```

Comment:
Why was the stop function remove? It's also being reference in few places, eg line 100